docs/docs/building-apps/02-app-mempool.md

```diff
@@ -35,29 +37,75 @@ Process proposal handles the validation of what is in a block, meaning that afte
-https://github.com/cosmos/cosmos-sdk/blob/bcff22a3767b9c5dd7d1d562aece90cf72e05e85/baseapp/baseapp.go#L906-L930
+https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-alpha2/baseapp/baseapp.go#L911-L935
 ```
 
-Like `PrepareProposal` this implementation is the default and can be modified by the application developer. 
+Like `PrepareProposal` this implementation is the default and can be modified by the application developer in [`app.go`](./01-app-go-v2.md):
 
-```go reference
-https://github.com/cosmos/cosmos-sdk/blob/main/simapp/app.go#L200-L203
+```go
+processOpt := func(app *baseapp.BaseApp) {
+	app.SetProcessProposal(app.DefaultProcessProposal())
+}
+
+baseAppOptions = append(baseAppOptions, processOpt)
 ```
 
 ## Mempool
 
 Now that we have walked through the `PrepareProposal` & `ProcessProposal`, we can move on to walking through the mempool. 
 
-There are countless designs that an application developer can write for a mempool, the core team opted to provide a simple implementation of a nonce mempool. The nonce mempool is a mempool that keeps transactions from an sorted by nonce in order to avoid the issues with nonces. 
+There are countless designs that an application developer can write for a mempool, the SDK opted to provide only simple mempool implementations.
+Namely, the SDK provides the following mempools:
+
+* [Sender Nonce Mempool](#sender-nonce-mempool)
+* [Priority Nonce Mempool](#priority-nonce-mempool)
+* [No-op Mempool](#no-op-mempool)
+
+The default SDK is a [Sender Nonce Mempool](#sender-nonce-mempool), but it can be replaced by the application developer in [`app.go`](./01-app-go-v2.md):
+
+```go reference
+nonceMempool := mempool.NewSenderNonceMempool()
+mempoolOpt   := baseapp.SetMempool(nonceMempool)
+baseAppOptions = append(baseAppOptions, mempoolOpt)
+```
+
+### Sender Nonce Mempool
 
+The nonce mempool is a mempool that keeps transactions from an sorted by nonce in order to avoid the issues with nonces. 
 It works by storing the transation in a list sorted by the transaction nonce. When the proposer asks for transactions to be included in a block it randomly selects a sender and gets the first transaction in the list. It repeats this until the mempool is empty or the block is full. 
 
-### Configurations
+It can configurable by the following parameters:
 
 #### MaxTxs
 
-Its an integer value that sets the mempool in one of three modes, bounded, unbounded, or disabled.
+It is an integer value that sets the mempool in one of three modes, *bounded*, *unbounded*, or *disabled*.
+
+* **negative**: Disabled, mempool does not insert new transaction and return early.
+* **zero**: Unbounded mempool has no transaction limit and will never fail with `ErrMempoolTxMaxCapacity`.
+* **positive**: Bounded, it fails with `ErrMempoolTxMaxCapacity` when `maxTx` value is the same as CountTx()
+
+#### Seed
+
+Set the seed for the random number generator used to select transactions from the mempool.
+
+### Priority Nonce Mempool
```

Comment:
We should open an issue about max txs here. I didn't realize we merged without adding this.